assembl/graphql/user.py

```diff
@@ -74,6 +86,39 @@ def mutate(root, args, context, info):
         with cls.default_db.no_autoflush as db:
             user.username_p = args.get('username')
             user.real_name_p = args.get('name')
+
+            # add uploaded image as an attachment to the user
+            image = args.get('image')
+            if image is not None:
+                filename = os.path.basename(context.POST[image].filename)
```

Comment:
You should probably add CRUD logic check on the Attachment model.

Reply:
done in https://github.com/assembl/assembl/pull/265/commits/0e9e51aee2799a172f3663a53425cd6050974104